packages/math-input/src/components/gesture-state-machine.ts

```diff
@@ -49,16 +49,16 @@ const defaultOptions: Options = {
 class GestureStateMachine {
     handlers: Handlers;
     options: Options;
-    swipeDisabledNodeIds: Partial<[Keys]>;
```

Comment:
Good catch.  I know KevinB raised the question of how we annotate array types: `Array<Key>` vs `Key[]`.  In my mind, I prefer `Array<Key>` even if the TS compiler will emit messages using the other array syntax, just because it'll be easier for us to spot this type of mistake (declaring a tuple type when we need `Array<>`). 

What do you think?